src/app/legacy/containers/SocialEmbed/sourceHelpers.js

```diff
@@ -27,6 +28,9 @@ export const getProviderFromSource = source => {
   if (source.match(/^https:\/\/www\.tiktok\.com/)) {
     return PROVIDERS.TIKTOK;
   }
+  if (source.match(/^https:\/\/www\.facebook\.com/)) {
```

Comment:
Are we always guranteed the embeds will start with `www.`? I'm also aware of the [`m.`](https://m.facebook.com/RickAstley/posts/545713756920775) subdomain for their mobile site, does that need to be supported?

Reply:
I think all of our social media providers have a `m.` or `mobile.` subdomain, which we don't currently support so I imagine it will be the case here too?

I can check with the Optimo team if that is true.

Reply:
I guess it can't hurt to check